x-pack/plugins/security_solution/cypress/tasks/login.ts

```diff
@@ -239,3 +239,8 @@ export const loginAndWaitForTimeline = (timelineId: string, role?: RolesType) =>
   cy.get('[data-test-subj="headerGlobalNav"]');
   cy.get(TIMELINE_FLYOUT_BODY).should('be.visible');
 };
+
+export const waitForPageWithoutDateRange = (url: string, role?: RolesType) => {
+  cy.visit(role ? getUrlWithRoute(role, url) : url);
+  cy.get('[data-test-subj="headerGlobalNav"]', { timeout: 120000 });
```

Comment:
Any reason this timeout is set so high?

Reply:
Not sure. What I did is took an existing function `loginAndWaitForPageWithoutDateRange` (it's above this one) and removed the `login` part from it to be able to implement the tests.

`loginAndWaitForPageWithoutDateRange` with this timeout was introduced by @XavierM along with the `"defaultCommandTimeout": 120000` in `cypress.json`. See here:

https://github.com/elastic/kibana/commit/1216b0f7cd2c0697393547432bee343ed33b96b5

I guess that's some legacy, because the current `"defaultCommandTimeout": 60000`